packages/backend/src/index.ts

```diff
@@ -161,5 +161,6 @@ if (process.env.ENABLE_AUTH_PROVIDER_MODULE_OVERRIDE !== 'true') {
 backend.add(import('@internal/plugin-dynamic-plugins-info-backend'));
 backend.add(import('@internal/plugin-scalprum-backend'));
 backend.add(import('@internal/plugin-licensed-users-info-backend'));
+backend.add(import('@backstage/plugin-user-settings-backend'));
```

Comment:
Hi @karthikjeeyar, none of our plugins has this at the moment.

But wdyt, can we have this enabled by default, but with an app-config opt-out option?

```yaml
user-settings:
  persistence: false
```

Which means the backend isn't loaded? Not sure if your change in `app/src/apis.ts` must know and handle this as well?